google/search/src/server.ts

```diff
@@ -0,0 +1,176 @@
+      }
+    },
+    required: ['search']
+  },
+  chat: false,
+  context: [],
+  credentials: [],
+  description: '',
+  export: [],
+  exportContext: [],
+  globalModelName: '',
+  globalTools: [],
+  jsonResponse: true,
+  maxTokens: 0,
+  modelName: 'gpt-4o-mini',
+  modelProvider: false,
+  name: '',
+  tools: [],
+  temperature: 0.2,
+  instructions: dedent(`
+  Given a search object with the following JSON schema: 
+
+  ${minify({
+  "title": "Search",
+  "type": "object",
+  "properties": {
+    "query": {
+      "type": "string",
+      "description": "The search query used"
+    },
+    "results": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "url": {
+            "type": "string",
+            "format": "uri",
+            "description": "URL of the web page"
+          },
+          "content": {
+            "type": "string",
+            "description": "Content of the web page at URL in markdown"
+          }
+        },
+        "required": ["url", "content"],
+        "additionalProperties": false
+      },
+      "description": "An array of search results"
+    }
+  },
+  "required": ["query", "results"],
+  "additionalProperties": false
+  })}
+
+  For all search results, silently select all the chunks of text from each result's content that: 
+  - answer the search query directly or provide additional information to support an answer to the search query
+  - contain enough context from the source text to complete the thought expressed by them in the original source text
+  
+  Then generate a single FocusedSearch object containing all of the selected chunks using the following JSON schema:
+
+  ${minify({
+  "title": "FocusedSearch",
+  "type": "object",
+  "properties": {
+    "query": {
+      "type": "string",
+      "description": "The search query used"
+    },
+    "results": {
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "url": {
+            "type": "string",
+            "format": "uri",
+            "description": "URL of the search result's web page"
+          },
+          "title": {
+            "type": "string",
+            "description": "Title of the search result's web page"
+          },
+          "relevantContent": {
+            "type": "array",
+            "items": {
+              "type": "string"
+            },
+            "description": "Chunks of text sourced from the web page at URL that answer the search query"
+          }
+        },
+        "required": ["url", "title", "relevantContent"],
+      },
+      "description": "An array of focused search results"
+    }
+  },
+  "required": ["query", "results"],
+  })}
+
+  Remove all results with no relevant content and deduplicate semantically equivalent relevant content before
+  responding with the minified JSON of the FocusedSearch object.
+  
+  Do not include any additional preamble or commentary.
```

Comment:
In knowledge, I had to prompt it to not use syntax highlighting (i.e. backticks surrounding the JSON), as it was adding them every now and then (not always)

Reply:
Yeah, I've been pumping the output through jq as I've tested this and haven't hit that problem yet. I think this line may be why